### ISSUE TITLE
Crashes with strange gdbus errors

### ISSUE THREAD

**Issue by tsjnachos117** [celluloid-player/celluloid#688]
Overview Description:
Celluloid sometimes crashes with a strange gdbus error.

Steps to Reproduce:
This but is not actually 100% reproducible. Some times this program works just fine, but only whenever the heck it feels like it. IDK if there's any rhyme or reason behind this.

Actual Results:
It crashes with no warning. On the terminal, it prints the following:
`GLib-GIO:ERROR:../glib/gio/gdbusconnection.c:4217:invoke_get_property_in_idle_cb: assertion failed: (error != NULL)
Bail out! GLib-GIO:ERROR:../glib/gio/gdbusconnection.c:4217:invoke_get_property_in_idle_cb: assertion failed: (error != NULL)
Aborted (core dumped)
`

Expected Results:
Not crashing, of course.

Version:
0.21. I just recompiled from the latest git, but that didn't help at all.

Additional Information:
I am running Arch GNU/linux with the latest updates. This program was not installed from the repos, but build from source. I did not use an AUR. I also custom-built mpv from source using the [mpv build script repo](https://github.com/mpv-player/mpv-build), also without using AUR.

Possibly related: when this program does work, the menu bar is hidden unless I enter and exit fullscreen. This isn't a major issue, and probably shouldn't be discussed to heavily on this particular thread, but I though I'd mention it just in case it somehow winds up being relevant.

**Comment by gnome-mpv** [comment]
Are you using KDE Connect? According to https://dev.getsol.us/T9889, it's triggering the crash.

**Comment by M86xKC** [comment]
Yes I can confirm when using KDE Connect and connecting to a phone, celluloid crashes with the same error.

Here is the same issue reported almost 1 month ago on reddit redd.it/pbvz59

Please note that this issue only happens for kdeconnect versions above 21.04 (21.04 works fine).

**Comment by OdinVex** [comment]
It's back, on Mint 21 for me. Killed KDEConnect, fine afterwards.

```
GLib-GIO:ERROR:../../../gio/gdbusconnection.c:4284:invoke_get_property_in_idle_cb: assertion failed: (error != NULL)
Bail out! GLib-GIO:ERROR:../../../gio/gdbusconnection.c:4284:invoke_get_property_in_idle_cb: assertion failed: (error != NULL)
Aborted (core dumped)
```
